components/camel-jpa/src/main/java/org/apache/camel/component/jpa/JpaEndpoint.java

```diff
@@ -185,6 +185,16 @@ protected String createEndpointUri() {
         return "jpa" + (entityType != null ? "://" + entityType.getName() : "");
     }
 
+    @Override
```

Comment:
I wonder why sonarcloud reports a problem that equals/hashCode needs to be overridden when they just call super?

Reply:
Sonar cares about additional [subclass fields and the needeed of add them in equals and hash code methods](https://rules.sonarsource.com/java/RSPEC-2160). In that case this is not necessary as equality is based on context name and endpoint Uri so additional fields dont need to be considered. However sonar points as issue.

Reply:
@orpiske `equals` and `hashCode` removed from last commit.